src/gui/screen_menu_esp_update.cpp

```diff
@@ -99,12 +99,12 @@ ScreenMenuESPUpdate::ScreenMenuESPUpdate()
     , menu(this, GuiDefaults::RectScreenBody - Rect16::Height_t(get_help_h()), &container)
     , header(this)
     , help(this, Rect16(GuiDefaults::RectScreen.Left(), uint16_t(GuiDefaults::RectScreen.Height()) - get_help_h(), GuiDefaults::RectScreen.Width(), get_help_h()), is_multiline::yes)
-    , firmware_set({ { //          { .address = BOOT_ADDRESS, .filename = "/boot_v1.7_with_flash_params.bin", .size = 0 },
-          { .address = USER_ADDRESS, .filename = "/user1.1024.new.2.bin", .size = 0 },
-          { .address = BLANK1_ADDRESS, .filename = "/blank.bin", .size = 0 },
-          { .address = BLANK2_ADDRESS, .filename = "/blank.bin", .size = 0 },
-          { .address = INIT_DATA_ADDRESS, .filename = "/esp_init_data_default_v08.bin", .size = 0 },
-          { .address = BLANK3_ADDRESS, .filename = "/blank.bin", .size = 0 } } })
+    , firmware_set({ { { .address = BOOT_ADDRESS, .filename = "/ESP/boot_v1.7.bin", .size = 0 },
+          { .address = USER_ADDRESS, .filename = "/ESP//user1.1024.new.2.bin", .size = 0 },
+          { .address = BLANK1_ADDRESS, .filename = "/ESP//blank.bin", .size = 0 },
+          { .address = BLANK2_ADDRESS, .filename = "/ESP//blank.bin", .size = 0 },
+          { .address = INIT_DATA_ADDRESS, .filename = "/ESP//esp_init_data_default_v08.bin", .size = 0 },
+          { .address = BLANK3_ADDRESS, .filename = "/ESP//blank.bin", .size = 0 } } })
```

Comment:
Why do you have the double slashes there? While it works (duplicate `/` are generally ignored), it shouldn't be needed.

Reply:
stupidity corrected